api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py

```diff
@@ -128,6 +128,50 @@ def test_call_environment_webhook_for_feature_state_change_with_previous_state_o
     assert data["timestamp"] == now_isoformat
 
 
+def test_call_environment_webhook_for_feature_state_change_with_changed_by_user_id(
+    mocker, environment, feature, identity, admin_user
+):
```

Comment:
Typing please

Reply:
Ahh, we should make pre-commit catch this